etl/steps/data/garden/unicef/2024-07-30/child_migration.excluded_countries.json

```diff
@@ -0,0 +1,35 @@
+[
```

Comment:
Why not keeping their aggregates?

Reply:
Currently, we show them and add the source as you describe. There are plans to change that, but I see no problem if you add at least some of them.

Reply:
Perfect, thank you! I'll do that then :)